connector_search_engine/models/se_index.py

```diff
@@ -214,3 +243,4 @@ def resynchronize_all_bindings(self):
     def delete_obsolete_item(self, item_ids):
         adapter = self._get_backend_adapter()
         adapter.delete(item_ids)
+        return f"Deleted ids : {item_ids}"
```

Comment:
```suggestion
        items = self.env["se.binding.todelete"].sudo().browse(item_ids)
        adapter = self._get_backend_adapter()
        adapter.delete(items.mapped("res_id"))
        items.unlink()
        return f"Deleted ids : {item_ids}"
```
The other option is that you add another method that calls this.
Also, would be nice to make some clarity on `item_ids`.